src/MyEventEmitter.js

```diff
@@ -1,14 +1,68 @@
+/* eslint-disable max-len */
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.events = {};
+  }
+
+  checkEventName = (eventName) => {
+    if (!this.events[eventName]) {
+      this.events[eventName] = [];
+    }
+  };
+
+  on(eventName, listener) {
+    this.checkEventName(eventName);
+    this.events[eventName].push(listener);
+  }
+
+  once(eventName, listener) {
+    const onceListener = (...args) => {
+      this.off(eventName, onceListener);
+      listener(...args);
+    };
+
+    this.on(eventName, onceListener);
+  }
+
+  off(eventName, listener) {
+    this.events[eventName] = this.events[eventName].filter(
+      (registeredListener) => registeredListener !== listener,
+    );
+  }
+
+  emit(eventName, ...args) {
+    this.events[eventName].forEach((listener) => {
+      listener(...args);
+    });
```

Comment:
The `emit` method also doesn't check if the event exists before trying to emit it. If the event doesn't exist, an error will be thrown. To fix this, you should add a check for the event's existence before trying to emit it.